i18n/en-US.properties

```diff
@@ -424,14 +424,16 @@ be.tasks.feed.headline.approval.currentUser = {user} assigned you an Approval Ta
 be.tasks.feed.headline.general = {user} assigned a Task
 # Comment headline for a general task assigned to the current user
 be.tasks.feed.headline.general.currentUser = {user} assigned you a Task
-# Label for an incomplete task
-be.tasks.feed.incompleteLabel = Awaiting Assignees
+# Label for a task in progress
+be.tasks.feed.inProgressLabel = In Progress
 # Label for button to expand flyout to see all task assignees
 be.tasks.feed.moreAssigneesLabel = See all assignees
 # Reject option for an approval task
 be.tasks.feed.rejectAction = Reject
 # Label for a rejected task
-be.tasks.feed.rejectedLabel = Task Rejected
+be.tasks.feed.rejectedLabel = Rejected
+# Label for the task status
+be.tasks.feed.statusLabel = Status:
```

Comment:
Looks good, my only comment is I _think_ when you use `:` you should put a variable after instead of concatenating the strings in code, because different languages might have different ways of treating this grammar